code_samples/map_removeAll_sample.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { Client, Predicates } = require('../lib/index');
```

Comment:
```suggestion
const { Client, Predicates } = require('hazelcast-client');
```

Reply:
Yes, my bad. Just wanted to see if it works